controllers/games-controller.js

```diff
@@ -0,0 +1,12 @@
+const { fetchAllCatogories } = require("../models/games-model");
+
+exports.getAllCatogories = (req, res, next) => {
+  fetchAllCatogories()
+    .then((data) => {
+      res.status(200).send(data);
```

Comment:
Please return this data on a key of categories so it looks like the data you can see in `endpoints.json`